src/main/business/applicationFacade.js

```diff
@@ -336,6 +338,27 @@ class ApplicationFacade{
         return this.userController.updateUser(data);
     }
 
+    async generateTask(data) {
+        validateParameters(data, {taskType: PRIMITIVE_TYPES.STRING});
+        switch (data.taskType) {
+            case GENERATED_TASK_TYPES.TAG_ANSWER:
+                return this.#generateTagAnswerTask(data)
+            default:
+                throw new EMSError(TASK_PROCESS_ERROR_MSGS.INVALID_TASK_TYPE(data.taskType), TASK_PROCESS_ERROR_CODES.INVALID_TASK_TYPE);
+        }
+    }
+
+    async completeGeneratedTask(data) {
+        validateParameters(data, {taskType: PRIMITIVE_TYPES.STRING});
+        switch (data.taskType) {
+            case GENERATED_TASK_TYPES.TAG_ANSWER:
+                console.log(data);
```

Comment:
Is this just a "mock" function?

If so please make it more abvious by adding an appropriate doc or something else

Reply:
this is outdated